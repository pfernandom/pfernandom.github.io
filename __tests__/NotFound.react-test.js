import React from 'react';
import renderer from 'react-test-renderer';
import ReactTestUtils from 'react-addons-test-utils'
import NotFound from '../src/components/error';


const rend = ReactTestUtils.createRenderer();
rend.render(<NotFound />);
const result = rend.getRenderOutput();

test('NotFound matches snapshot', () => {
	const component = renderer.create(
		<NotFound/>
	);
	let tree = component.toJSON();
	expect(tree).toMatchSnapshot();
});



