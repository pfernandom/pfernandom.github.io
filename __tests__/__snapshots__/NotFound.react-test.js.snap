exports[`test NotFound matches snapshot 1`] = `
<h1>
  404.. This page is not found! :(
</h1>
`;
